app/libraries/scim_patch_operation.rb

```diff
@@ -2,57 +2,123 @@
 
 # Parse One of "Operations" in PATCH request
 class ScimPatchOperation
```

Comment:
```suggestion
class ScimPatch
```
にすると中でop持っても別になんとも思わない気がした

Reply:
すでに ScimPatch を作ってしまっているんですよね。。。
ScimPatch の責務をもう少し増やして、現在 struct で作っているものを各種 Operation クラスにするというのは有りかも。と思ってます。

ただ、一方で最小限で実装を進めるために、 現在のScimPatchOperation を/Users 用のクラスと /Groups 用のクラスに分けて実装することも考えていて、それと合わせて検討するのでいったんこのままでお願いします。